truffle/migrations/2_deploy_hello_blockchain_world.js

```diff
@@ -1,8 +1,7 @@
 const HelloBlockchainWorld = artifacts.require("./HelloBlockchainWorld.sol");
 
 module.exports = (deployer) => {
-  console.log("Deploying `HelloBlockchainWorld.sol` ...");
-  deployer.deploy(HelloBlockchainWorld, { overwrite: false }).then(() => {
```

Comment:
Why did you modify this? Is there any inconvenient in keeping this logs?

Reply:
answered above.